### DISCUSSION TITLE
Unable to locate package python-serial

### DISCUSSION THREAD

**Discussion by tlzfighting** [contiki-ng/contiki-ng#2302] [Q&A]
When I use the compile command      "sudo apt install python-serial",
The following error occurred,How should this situation be resolved?
![图片](https://user-images.githubusercontent.com/95647449/204116183-010b0870-e8d4-4b54-b26d-ebb8bb1ed171.png)

**Comment by pjonsson** [comment]
This is fixed by #2326.
